dymos/grid_refinement/test/test_error_estimation.py

```diff
@@ -58,13 +58,15 @@ def _run_brachistochrone(self, transcription_class=dm.Radau, control_type='contr
         p['traj0.phase0.t_initial'] = 0.0
         p['traj0.phase0.t_duration'] = 2.0
 
-        p['traj0.phase0.states:x'] = phase.interpolate(ys=[0, 10], nodes='state_input')
-        p['traj0.phase0.states:y'] = phase.interpolate(ys=[10, 5], nodes='state_input')
-        p['traj0.phase0.states:v'] = phase.interpolate(ys=[0, 9.9], nodes='state_input')
+        p.set_val('traj0.phase0.states:x', phase.interp('x', [0, 10]), units='m')
+        p.set_val('traj0.phase0.states:y', phase.interp('y', [10, 5]), units='m')
+        p.set_val('traj0.phase0.states:v', phase.interp('v', [0, 5]), units='m/s')
```

Comment:
Changes here too. Wonder why you didn't need to change any asserts.

Reply:
Same, final velocity is unconstrained.